ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/423__model_serving_customruntimes.robot

```diff
@@ -0,0 +1,83 @@
+*** Settings ***
+Documentation     Collection of tests to validate the model serving stack for Large Language Models (LLM)
+Resource           ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource
+Resource          ../../../Resources/Page/ODH/ODHDashboard/ODHDashboardSettingsRuntimes.resource
+Resource          ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/DataConnections.resource
+Resource          ../../../Resources/Page/ODH/ODHDashboard/ODHModelServing.resource
+Suite Setup       Custom Serving Runtime Suite Setup
+Suite Teardown    Custom Serving Runtime Suite Teardown
+
+
+*** Variables ***
+${RESOURCES_DIRPATH}=        ods_ci/tests/Resources/Files
+${OVMS_RUNTIME_FILEPATH}=    ${RESOURCES_DIRPATH}/ovms_servingruntime.yaml
+${UPLOADED_OVMS_DISPLAYED_NAME}=    ODS-CI Custom OpenVINO Model Server
+${PRJ_TITLE}=    CustomServingRuntimesProject
+${PRJ_DESCRIPTION}=    ODS-CI DS Project for testing of Custom Serving Runtimes
+${MODEL_SERVER_NAME}=    ODS-CI CustomServingRuntime Server
+
+
+*** Test Cases ***
+Verify RHODS Admins Can Import A Custom Serving Runtime Template By Uploading A YAML file
+    [Tags]    Smoke    ODS-2276
+    Open Dashboard Settings    settings_page=Serving runtimes
+    Upload Serving Runtime Template    runtime_filepath=${OVMS_RUNTIME_FILEPATH}
+    Serving Runtime Template Should Be Listed    displayed_name=${UPLOADED_OVMS_DISPLAYED_NAME}
+
+Verify RHODS Admins Can Delete A Custom Serving Runtime Template
+    [Tags]    Smoke    ODS-2279
+    [Setup]    Create Test Serving Runtime Template If Not Exists
+    Open Dashboard Settings    settings_page=Serving runtimes
+    Delete Serving Runtime Template    displayed_name=${UPLOADED_OVMS_DISPLAYED_NAME}
+    ...    press_cancel=${TRUE}
+    Delete Serving Runtime Template    displayed_name=${UPLOADED_OVMS_DISPLAYED_NAME}
```

Comment:
Wouldn't it be better to make these two tests independent of each other right away? we risk having the same situation as ds projects otherwise

Reply:
they are independent

Reply:
whoops, missed that